projects/addon-doc/components/code/index.less

```diff
@@ -50,6 +50,9 @@
     justify-content: center;
     align-items: center;
     flex-direction: row-reverse;
+    padding: 0.375rem;
```

Comment:
Can we just use whiteblock appearance and a bigger size?

Reply:
fixed